types/react-bootstrap/lib/Accordion.d.ts

```diff
@@ -1,17 +1,19 @@
 import * as React from 'react';
 import { Sizes } from 'react-bootstrap';
 
-declare class Accordion extends React.Component<AccordionProps> { }
-declare namespace Accordion { }
+declare namespace Accordion {
+    export interface AccordionProps extends React.HTMLProps<Accordion> {
```

Comment:
maybe also include an alias, `export type Props = AccordionProps`, so that you can reference as `Accordion.Props`?

Reply:
The previous version `0.x.x` exported props in the current format: `NameOfTheComponent` + `Props` giving `NameOfTheComponentProps`. People were used to use that so exporting an alias seems not really necessary I think.